File: Selenium/Activities/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Firefox() as driver:
    # Navigate to the URL
    driver.get("https://training-support.net/webelements/dynamic-controls")

    
    print("Page title is: ", driver.title)
    
	
    textbox = driver.find_element(By.ID, "textInput")
    
    print("Textbox is enabled: ", textbox.is_enabled())
    
    driver.find_element(By.ID, "textInputButton").click()
    
    print("Textbox is enabled: ", textbox.is_enabled())

File: Selenium/Activities/Activity7.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;


public class Activity7 {
	
	public static void main(String[] args) {
        
        WebDriver driver = new FirefoxDriver();
        
        driver.get("https://training-support.net/webelements/dynamic-controls");
        
        System.out.println("Page title: " + driver.getTitle());

        // Find the text field
        WebElement textbox = driver.findElement(By.id("textInput"));
        // Check if it is enabled
        System.out.println("Input field is enabled: " + textbox.isEnabled());
        
        driver.findElement(By.id("textInputButton")).click();
        
        System.out.println("Input field is enabled: " + textbox.isEnabled());
        
        
        textbox.sendKeys("Example text");
        System.out.println(textbox.getDomProperty("value"));

        // Close the browser
        driver.quit();
    }

}
